docs/development/codeguide.rst

```diff
@@ -168,11 +168,11 @@ Coding Style/Conventions
   addition, these checks also enforce `isort <https://pycqa.github.io/isort/>`_ to sort
   the module imports and a large set of style-checks supported by ruff_.
 
-  * We provide a ``pre-commit`` hook which automatically enforces and fixes (whenever
-    possible) the coding style, see :ref:`pre-commit` for details on how to set up and
-    use this. We note that the particular set of |PEP8| and style-related checks that are
-    used in Astropy do not need to be used in affiliated packages. In particular, the
-    set of ruff_ checks is not required for affiliated packages.
+  * We provide a ``pre-commit`` configuration which automatically enforces and fixes
+    (whenever possible) the coding style, see :ref:`pre-commit` for details on how to
+    set up and use this. We note that the particular set of |PEP8| and style-related
+    checks that are used in Astropy do not need to be used in affiliated packages. In
+    particular, the set of ruff_ checks is not required for affiliated packages.
```

Comment:
Why does the `astropy` developer documentation talk about affiliated packages at all?

Reply:
context: https://github.com/astropy/astropy/issues/14737

Reply:
On one hand, to the readers of this document the talk about affiliated packages is needless clutter, especially for first-time contributors. On the other hand, if you wanted to look up the guidelines for affiliated packages then surely you wouldn't look for them here. You'd look up the [actual guidelines](https://www.astropy.org/affiliated/#becoming-an-affiliated-package) instead.

Reply:
Yes I think that it makes more sense to put this disclaimer there. WDYT @pllim ?

Reply:
I think the thing that comes closest is the package template that's linked: https://github.com/OpenAstronomy/packaging-guide
it contains a `.pre-commit-config.yaml` but doesn't include ruff.